scripts/publish.sh

```diff
@@ -104,9 +104,4 @@ echo "Tagging and publishing release"
 yarn -s --ignore-scripts publish --access=public
 
 echo "Pushing git commit and tag"
-git push --follow-tags
-
-echo "Publish successful!"
-echo ""
-
-create_github_release
```

Comment:
intentionally rm'd? and is the --follow-tags not needed?

Reply:
nope lost in copy paste, added back